test/framework/e2e.go

```diff
@@ -26,6 +26,7 @@ import (
 const (
 	defaultClusterConfigFile = "cluster.yaml"
 	defaultClusterName       = "eksa-test"
+	clusterNameVar           = "T_CLUSTER_NAME"
 	JobIdVar                 = "T_JOB_ID"
```

Comment:
Does this need to be defined anywhere in codebuild or is it good to go?

Reply:
I does not need to be defined. The default value is on line 297 below